### PR TITLE
Fix profil2

### PR DESCRIPTION
**Pull request by BBriset**
First deletation and modifications are from duplicate folder on the dev

### CODE REVIEW

File: src/pages/SuppliesModal/style.jsx [hunk 1]
```diff
@@ -11,8 +15,22 @@ const SSuppliesModal = styled.main`
   }
   .basketIcon {
     height: 8vh;
-    margin-left: 30vw;
-    padding-top: 20px;
+    margin-top: 3vh;
+  }
+
```
Comment (HeloiseFerla) [inline src/pages/SuppliesModal/style.jsx:20]: empty line

File: src/pages/SuppliesModal/index.jsx [hunk 2]
```diff
@@ -57,19 +57,21 @@ export default function SuppliesModal({
...
-            }}
-          />
+          <div className="order">
+            <MainButton
+              content="Passer la commande"
+              clickCallback={() => {
+                sendData();
+              }}
+            />
+
```
Comment (HeloiseFerla) [inline src/pages/SuppliesModal/index.jsx]: empty line

File: src/pages/Supplies/style.jsx [hunk 3]
```diff
@@ -34,13 +33,19 @@ const SSupplies = styled.main`
     }
```
Comment (HeloiseFerla) [inline src/pages/Supplies/style.jsx]: empty line

File: src/pages/Supplies/style.jsx [hunk 4]
```diff
@@ -2,9 +2,8 @@ import styled from 'styled-components';
 
 const SSupplies = styled.main`
   .container {
-    width: 80vw;
+    min-width: 90vw;
     display: block;
-    margin: auto;
   }
```
Comment (HeloiseFerla) [inline src/pages/Supplies/style.jsx]: empty line

**Review by HeloiseFerla — APPROVED** [review]

**Review by HeloiseFerla — CHANGES_REQUESTED** [review]
empty lines

**Review by HeloiseFerla — DISMISSED** [review]

**Review by Hacene010 — DISMISSED** [review]

**Review by HeloiseFerla — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**